src/ApolloHooksTypes.re

```diff
@@ -58,3 +60,14 @@ let errorPolicyToJs = errorPolicy =>
   | Ignore => "ignore"
   | All => "all"
   };
+
```

Comment:
Can someone check these type definitions more closely? @jfrolich, you have more knowledge in `graphql_ppx_re` and its types, appreciate a quick look! 😄

Reply:
Looks good. Only would I rename `'composedFunction` to `'hookReturnType` as it's also a return type.